components/atoms/modal-body/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import styles from './styles.css'
+
+const ModalBody = ({ className, children }) => (
```

Comment:
Do you think we need this atom? Can it maybe be renamed to `block` or something like that? Because it's very generic (a `div` with a class and children), but with this name no other component will use it.

Reply:
We removed the prefix modal and renamed these components that are specific to modal just to header, body and footer.
And to be clearer we moved these components inside to modal directory and export them